NewSpring/ApollosProject
# [DEV-4176] Add App Image

## DESCRIPTION

### What does this PR do, or why is it needed?
This PR overrides the `pickBestImage` function from core so that we can choose the image specifically for the app if there is one.

### How do I test this PR?
I set up `alpha-rock` so that General Content type content channels have an "App Image" attribute. Set an app image on a piece of content and make sure that it shows up in the app. 

## TODO


## REVIEW


**Manual QA**


---

> The purpose of PR Review is to _improve the quality of the software._
